### ISSUE TITLE
UDT Congestion Control?

### ISSUE THREAD

**Issue by fwflunky** [SLikeSoft/SLikeNet#74]
Hello, how to disable packet resend? Sometimes when i send a big packet to the server, raknet starts resending it many times . How to disable this feature?
![image](https://user-images.githubusercontent.com/38854160/161322465-0651a598-a9c2-488c-8b73-d09ecb1fbd93.png)

**Comment by Mellnik** [comment]
I don't think it resends it. It just splits it by the MTU. How many bytes is your data?

**Comment by fwflunky** [comment]
> I don't think it resends it. It just splits it by the MTU. How many bytes is your data?

~5000
